framework/requirements.txt

```diff
@@ -42,6 +42,7 @@ greenlet==2.0.2
 grpc-google-iam-v1==0.12.3
 grpcio==1.58.0
 hiredis==2.2.3
+httpx==0.26.0
```

Comment:
This PR will be blocked until the merging of:
- https://github.com/wazuh/wazuh/issues/20748